tutorial/getting-started-aaos.md

```diff
@@ -0,0 +1,78 @@
+This tutorial guides you through basic operations that you can do with an application based on an Android Automotive OS (AAOS) image. 
+
+In this tutorial, we will be focusing on the following tasks:
+- Creating an application using an AAOS image.
+- Stream the application you created with a specific screen resolution.
+- Set the temperature of the automotive using the Vehicle Hardware Abstraction Layer (VHAL) controls.
```

Comment:
Should we keep the same tense for all three points here? i.e. "Create" instead of "Creating"?

Reply:
Thanks, that's a good point. I made it consistent but used it the other way around because it reads better in the context.